irksome/getForm.py

```diff
@@ -156,19 +156,23 @@ def getForm(F, butch, t, dt, u0, bcs=None):
                 gcur = replace(gfoo, {t: t + c[i] * dt})
                 try:
                     gdat = interpolate(gcur, V)
-                except NotImplementedError:
+                    gmethod = lambda g: gdat.interpolate(g)
+                except:  # noqa: E722
```

Comment:
# noqa here (and below) because `interpolate` doesn't always throw a `NotImplementedError` when it isn't implemented

Reply:
Can you provide an example/report a Firedrake bug, this is kind of annoying for you.